frontend/assets/javascripts/src/main.js

```diff
@@ -26,7 +26,8 @@ require([
     'src/modules/patterns',
     'src/modules/memstatus',
     'src/modules/faq',
-    'src/modules/landingBundles'
+    'src/modules/landingBundles',
+    'src/modules/bundlesLanding'
```

Comment:
Aha, yeah that is a bit ridiculous I agree. Thing is, I can't think of a clearer name for this, it *is* the bundles landing page 🙃.